CONTRIBUTING.md

```diff
@@ -38,57 +38,46 @@ Finally, set the upstream remote to the official Spyder repo with:
 $ git remote add upstream https://github.com/spyder-ide/spyder.git
 ```
 
+
 ### Creating a conda environment or virtualenv
 
-If you use Anaconda you can create a conda environment with the following commands:
+If you use Anaconda you can create a conda environment and install the necessary dependencies as follows.
```

Comment:
```suggestion
If you use a Conda-based distribution (Anaconda, etc), you can create an environment and install the necessary dependencies as follows:
```

* Clarify that this applies to any Conda-based distribution (Anaconda, Miniconda, Miniforge, etc), not just Anaconda proper
* Add missing comma
* Use colon as ending punctuation to introduce next block